### PR TITLE
fix a test, fixes #5068

### LINKED ISSUE #5068: Have all ios tests use sample-apps

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

**Comment by Jonahss** [comment]
not for 1.4

File: test/functional/ios/file-movement-specs.js [hunk 1]
```diff
@@ -88,10 +92,10 @@ describe('file movements - pullFile and pushFile @skip-real-device', function ()
             basePath = path.resolve(simRoots[0], 'Applications');
           }
           basePath = basePath.replace(/\s/g, '\\ ');
-          var findCmd = 'find ' + basePath + ' -name "TestApp.app"';
+          var findCmd = 'find ' + basePath + ' -name "'+appName+'"';
```
Comment (jlipps) [inline test/functional/ios/file-movement-specs.js]: spacing

File: test/functional/ios/file-movement-specs.js [hunk 2]
```diff
@@ -109,7 +113,7 @@ describe('file movements - pullFile and pushFile @skip-real-device', function ()
     });
 
     it('should be able to fetch a file from the app directory', function (done) {
-      var arg = path.resolve('/TestApp.app', fileName);
+      var arg = path.resolve('/'+appName, fileName);
```
Comment (jlipps) [inline test/functional/ios/file-movement-specs.js]: spacing

**Comment by Jonahss** [comment]
fix'd spacing.

**Comment by jlipps** [comment]
probably don't want to commit testRun.txt

**Comment by Jonahss** [comment]
definitely do not

**Pull request merged**